.github/workflows/cgroup2.yaml

```diff
@@ -29,7 +32,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Boot Fedora
-        run: |
+        un: |
```

Comment:
this change and the r at the end of the file appear to be errant

Reply:
Thanks, took care of it.